### ISSUE TITLE
Feature Request: Tie to next Transition

### ISSUE THREAD

**Issue by chrisspiegl** [exeldro/obs-downstream-keyer#2]
Thank you for making this plugin available. It's incredible to see this added to OBS.

I come from the world of ATEM Mini switchers which also have Downstream Keyers and wondered if it would be possible to add a "TIE" feature to this plugin?

Essentially it would mean that there is a button (probably somewhere next to the pause button) which is either enabled or disabled labeled "TIE".

When TIE is not active: upon clicking on one of the scenes in the DSK it would immediately go in effect.

And when TIE is activated: upon clicking one of the scenes in the DSK it only get's "loaded" as the next active DSK scene and would then change with the next 'global transition' meaning a transition between two scenes.

Ideally this would also make it so that the 'pause' button would also react together with the TIE and upon clicking pause when TIE is active it would take the DSK off air upon switching scenes.

Just wanted to put this out there and see if this would be possible.

Thanks again and I am looking forward to see what will become of this plugin (in my opinion it should be added as a standard part of OBS 👍).

**Comment by exeldro** [comment]
added with commit https://github.com/exeldro/obs-downstream-keyer/commit/72e19dd5d7e62bd47f58b1445e3c3596bca617a3
you can get a test build here: https://github.com/exeldro/obs-downstream-keyer/actions/runs/996623932

**Comment by chrisspiegl** [comment]
Very intersting implementation. Incredible that you were able to do it so quickly.

Would it be possible to also make the DSK Tie button attatchable to a shortcut or controllable via webhooks?

**Comment by exeldro** [comment]
yes, test version with hotkeys building here: https://github.com/exeldro/obs-downstream-keyer/actions/runs/1001890327
